src/utils/validators.ts

```diff
@@ -35,9 +36,9 @@ export const validateLuhn = (cardNumber: string) => {
   );
 };
 
-export const getCardNumberError = (cardNumber: string, { errorMessages = {} as any } = {}) => {
```

Comment:
`errorMessages` as at my latest push is now typed as a dictionary, basically `{[key: string]: string}`.